test/e2e/suite_test.go

```diff
@@ -709,8 +709,8 @@ func applyPollingDefaults() {
 	polling.reaperReady.timeout = 10 * time.Minute
 	polling.reaperReady.interval = 15 * time.Second
 
-	polling.medusaBackupDone.timeout = 5 * time.Minute
-	polling.medusaBackupDone.interval = 5 * time.Second
+	polling.medusaBackupDone.timeout = 10 * time.Minute
+	polling.medusaBackupDone.interval = 15 * time.Second
```

Comment:
This is not directly related, but this seems to cause frequent CI errors recently.